src/hooks/useFetchRecommendationPod.ts

```diff
@@ -19,13 +19,13 @@ const useFetchRecommendationPod = (
       );
       const recommendationPodResults = {};
 
-      responses.forEach(({ response }) => {
+      responses.forEach(({ response }, index) => {
         const { pod, results } = response;
         if (pod?.id) {
           recommendationPodResults[pod.id] = results?.map((item: Item) => ({
             ...item,
             id: item?.data?.id,
-            section: pod.id,
```

Comment:
Are we using the pod id for anything? If so, we might want to keep it around

Reply:
As far as I can tell, we don't use it anywhere